torch/_dynamo/config.py

```diff
@@ -442,6 +442,10 @@ def default_debug_dir_root():
 # mutated after the print statement.
 reorderable_logging_functions: Set[Callable[[Any], None]] = set()
 
+# A set of methods that will be ignored while tracing,
+# to prevent graph breaks.
+ignore_logger_methods: Set[Callable] = set()
```

Comment:
Add a comment detailing that functions that go here should be in the form `logging.Logger.*` (I see in your test that `logger_obj.*` doesn't do anything)